src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -47,6 +47,11 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
  */
 @property (readonly, nonatomic, getter=isRunning) BOOL running;
 
+/**
+ * The ID assigned to this controller at creation time.
+ */
+@property (readonly, nonatomic) NSUUID * UUID MTR_NEWLY_AVAILABLE;
```

Comment:
Better to name as uniqueIdentifier? That way it is not tied to the typename in it ?

Reply:
Will do.

Reply:
Done.